recipes/aloe/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "aloe" %}
+{% set version = "0.1.12" %}
+{% set sha256sum = "2671488ffbde1850dcfd3d45c21ff2081a42dcbd0f34ac5dede66b8c1d5c2b82" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools_scm
+
+  run:
+    - python
+    - nose
+    - future
+    - gherkin-official >=4,<5
+    - nose
+    - repoze.lru
+
+test:
+  imports:
+    - aloe
+
+about:
+  home: https://github.com/aloetesting/aloe
+  license: GPL-3.0
+  license_file: '{{ environ["RECIPE_DIR"] }}/COPYING'
```

Comment:
Please ask upstream to include this in the `MANIFEST.in` and xref the issue/PR in a comment in the `meta.yaml`. Might also nudge them to fill out the license file with the full license.